x-pack/test/functional/apps/saved_objects_management/import_saved_objects_between_versions.ts

```diff
@@ -34,35 +44,31 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should be able to import 7.13 saved objects into 8.0.0', async function () {
-      await retry.tryForTime(10000, async () => {
-        const existingSavedObjects = await testSubjects.getVisibleText('exportAllObjects');
-        // Kibana always has 1 advanced setting as a saved object
-        await expect(existingSavedObjects).to.be('Export 1 object');
-      });
+      const initialObjectCount = await getExportCount();
+
       await PageObjects.savedObjects.importFile(
         path.join(__dirname, 'exports', '_7.13_import_saved_objects.ndjson')
       );
       await PageObjects.savedObjects.checkImportSucceeded();
       await PageObjects.savedObjects.clickImportDone();
-      const importedSavedObjects = await testSubjects.getVisibleText('exportAllObjects');
-      // verifying the count of saved objects after importing .ndjson
-      await expect(importedSavedObjects).to.be('Export 87 objects');
+      await PageObjects.savedObjects.waitTableIsLoaded();
+
+      const newObjectCount = await getExportCount();
+      expect(newObjectCount - initialObjectCount).to.eql(86);
     });
 
     it('should be able to import alerts and actions saved objects from 7.14 into 8.0.0', async function () {
-      await retry.tryForTime(10000, async () => {
-        const existingSavedObjects = await testSubjects.getVisibleText('exportAllObjects');
-        // Kibana always has 1 advanced setting as a saved object
-        await expect(existingSavedObjects).to.be('Export 87 objects');
-      });
+      const initialObjectCount = await getExportCount();
+
       await PageObjects.savedObjects.importFile(
         path.join(__dirname, 'exports', '_7.14_import_alerts_actions.ndjson')
       );
       await PageObjects.savedObjects.checkImportSucceeded();
       await PageObjects.savedObjects.clickImportDone();
-      const importedSavedObjects = await testSubjects.getVisibleText('exportAllObjects');
-      // verifying the count of saved objects after importing .ndjson
-      await expect(importedSavedObjects).to.be('Export 110 objects');
+      await PageObjects.savedObjects.waitTableIsLoaded();
+
+      const newObjectCount = await getExportCount();
+      expect(newObjectCount - initialObjectCount).to.eql(23);
```

Comment:
We need a comment explaining why there are 23 additional documents before and after the import.